disnake/guild.py

```diff
@@ -2059,6 +2106,19 @@ async def edit(
                 else:
                     features.discard("INVITES_DISABLED")
 
+            if raid_alerts_enabled is not MISSING:
+                if community is not MISSING:
+                    raise ValueError(
```

Comment:
missing in `ValueError` docs

Reply:
0925d3568b45092d894f46a2d29ed447a9b033c7, feel free to suggest anything if the change isn't clear